src/python/pants/backend/python/lint/isort/subsystem.py

```diff
@@ -9,7 +9,7 @@ class Isort(PythonToolBase):
     """The Python import sorter tool (https://timothycrosley.github.io/isort/)."""
 
     options_scope = "isort"
-    default_version = "isort>=4.3.21,<4.4"
+    default_version = "isort>=5.0.0,<6.0"
```

Comment:
Given the width of the previous range (and the fact that we don't have a lockfile here), should this be `isort>=5.0.0,<5.1`?

Reply:
+1

Reply:
Actually, looks like 5.1.x is the newest. So `>=5.1.4,<5.2`: https://github.com/timothycrosley/isort/blob/develop/CHANGELOG.md

Reply:
It seems that the author is doing the right thing and not introducing breaking changes in 5.x release (when he introduced breaking changes he moved from 4.x to 5.x) hence my choice to allow all 5.x release.
Since there are changes going on there in recent weeks.